backend/python/utils/geocoding.py

```
@@ -279,6 +279,66 @@ def get_coordinates_from_nominatim_sync(location_string: str) -> Optional[Dict[s
+            response.raise_for_status()
+
+        with _nominatim_lock:
+            _last_nominatim_call_time = time.time()
+
+        data = response.json()
+        display_name = data.get("display_name") if isinstance(data, dict) else None
+        if display_name:
+            logger.info(f"Reverse geocoded coordinates to: {display_name[:60]}...")
+        return display_name
+
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)